src/accordion.js

```diff
@@ -39,7 +45,7 @@ Accordion.prototype.findTriggers = function() {
 };
 
 Accordion.prototype.setAria = function(trigger, index) {
-  var contentID = 'content-' + index;
+  var contentID = this.opts.contentPrefix + '-' + 'content-' + index;
```

Comment:
Ok, I update this. I added a check to see if there's already IDs set, and if so, use those. I think it's convenient to not have to worry about setting them and let the library do it programmatically (as in the case of a really long list, like the glossary). But good call on not changing things if they already exist.